src/nbl/asset/interchange/CGLTFLoader.cpp

```diff
@@ -1463,8 +1543,11 @@ namespace nbl
 									vertexInputParams.attributes[attributeID].relativeOffset = 0;
 								};
 
-								setOverrideBufferBinding(overrideSkinningBuffers.jointsAttributes, cpuMeshBuffer->getJointIDAttributeIx());
-								setOverrideBufferBinding(overrideSkinningBuffers.weightsAttributes, cpuMeshBuffer->getJointWeightAttributeIx());
+								cpuMeshBuffer->setJointIDAttributeIx(SAttributes::JOINTS_ATTRIBUTE_LAYOUT_ID);
+								cpuMeshBuffer->setJointWeightAttributeIx(SAttributes::WEIGHTS_ATTRIBUTE_LAYOUT_ID);
+
+								setOverrideBufferBinding(overrideSkinningBuffers.jointsAttributes, SAttributes::JOINTS_ATTRIBUTE_LAYOUT_ID);
+								setOverrideBufferBinding(overrideSkinningBuffers.weightsAttributes, SAttributes::WEIGHTS_ATTRIBUTE_LAYOUT_ID);
 							}
 
 							skinningEnabled = true;
```

Comment:
get rid of the lambda below, and address the TODO

Reply:
also why are you setting `skinningEnabled=true` always !?

Reply:
what? read the code please, this is enabled after override buffers creation when you know that skinning is enabled!

Reply:
by default `skinningEnabled` is off

Reply:
cut down the code in buffer repacking and maybe that lambda will fit on a single portrait 4k screen